portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase

```diff
@@ -103,6 +103,54 @@ definition {
 		}
 	}
 
+	@description = "LPS-147851. Assert the order button can display the correct icons"
+	@priority = "4"
+	test OrderButtonCanDisplayCorrectIcons {
+		task ("And clicks on the order button") {
+			Click(
+				key_filter = "Order",
+				locator1 = "ClaySamplePortlet#DISPLAY_CONTEXT_FILTER_OR_ORDER",
+				value1 = "Order");
+		}
+
+		task ("And selects the descending order") {
+			Click(
+				key_item = "Descending",
+				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM",
+				value1 = "Descending");
+		}
+
+		task ("Then the order icon will be displayed accordingly") {
+			Navigator.gotoNavTab(navTab = "Management Toolbars");
+
+			AssertElementPresent(
+				key_text = "order-list-down",
+				locator1 = "ClaySamplePortlet#DISPLAY_CONTEXT_ANY_ICON");
+		}
+
+		task ("And clicks on the order button") {
+			Click(
+				key_filter = "Order",
+				locator1 = "ClaySamplePortlet#DISPLAY_CONTEXT_FILTER_OR_ORDER",
+				value1 = "Order");
+		}
+
+		task ("And when selects the ascending order") {
+			Click(
+				key_item = "Ascending",
+				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM",
+				value1 = "Ascending");
+		}
+
+		task ("Then the order icon will be displayed accordingly") {
+			Navigator.gotoNavTab(navTab = "Management Toolbars");
+
+			AssertElementPresent(
+				key_text = "order-list-up",
+				locator1 = "ClaySamplePortlet#DISPLAY_CONTEXT_ANY_ICON");
```

Comment:
Instead of selenium/html path assertions, we could try using ocular for these assertions instead if the main goal of the test is to prove that icons are displayed correctly. ie: does not regress from what is currently displayed.